_studio/shared/umc/codec/h264_dec/src/umc_h264_au_splitter.cpp

```diff
@@ -40,9 +40,16 @@ SeiPayloadArray::SeiPayloadArray()
 SeiPayloadArray::SeiPayloadArray(const SeiPayloadArray & payloads)
 {
     size_t count = payloads.GetPayloadCount();
-    for (size_t i = 0; i < count; i++)
+    if (count == 0)
     {
-        AddPayload(payloads.GetPayload(i));
+        SeiPayloadArray();
```

Comment:
Here u just create temporary `SeiPayloadArray`. Maybe u're trying delegate constructor here? Delegating constructors works in the constructor's [initialization list](https://en.cppreference.com/w/cpp/language/initializer_list#Delegating_constructor) only.

Reply:
Hi @dmitry-gurulev 
I just want to call default constructor if count equals to zero.

Reply:
@LukeZheZhu here you just create temporary nameless object which will be destroyed at the end of scope. But as I said before, in my understanding you don't need to do anything. Which real problem you are trying to solve?

Reply:
The kk reports the m_payloads will uninitialized in this function.

Reply:
it is a ```std::vector``` it will be properly initialized by default. Static analysis reports false positive alert. No changes required